ext/pcntl/tests/pcntl_waitid.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+pcntl_waitid()
+--EXTENSIONS--
+pcntl
+posix
+--SKIPIF--
+<?php
+    if (!function_exists('pcntl_waitid')) die('skip pcntl_waitid unavailable');
+?>
+--FILE--
+<?php
+$pid = pcntl_fork();
+if ($pid == -1) {
+    die("failed");
+} else if ($pid) {
+    // test invalid arguments
+    var_dump(@pcntl_waitid(-2, $pid, $siginfo, WSTOPPED));
```

Comment:
would be nice to see related error messages also trying bad values for last argument.

Reply:
I agree, I would not suppress the warnings here. Worst case we can standardize them ourselves depending on what the errno is https://pubs.opengroup.org/onlinepubs/7908799/xsh/waitid.html

Reply:
Added checks for error messages, thanks.

@devnexen Bad value for last argument test was already there (WNOHANG by itself is a bad value for flags, a one of WEXITED, WSTOPPED, WCONTINUED is mandatory). I added a comment to the test as this might not be obvious.

Reply:
Sure but I was thinking about something more "silly" e.g. `PHP_INT_MAX`.

Reply:
Okay, added a couple of those as well.